charts/velero/README.md

```diff
@@ -87,8 +87,13 @@ If a value needs to be added or changed, you may do so with the `upgrade` comman
 ```bash
 helm upgrade vmware-tanzu/velero <RELEASE NAME> --reuse-values --set configuration.backupStorageLocation[0].provider=<NEW PROVIDER>
 ```
+## Upgrading Chart
 
-## Upgrading
+### Upgrading to 6.0.0
+
+This version removes the `nodeAgent.privileged` field, you should use `nodeAgent.containerSecurityContext.privileged` instead
```

Comment:
Do you think we should create another doc, such as [BREAKING_CHANGES.md](https://github.com/elastic/helm-charts/blob/main/BREAKING_CHANGES.md) to indicate which versions break the changes rather than write it down in the README.md?

Reply:
I personally prefer to have it all in the README so people don't miss it or have to click on another link to get to critical information.

Reply:
What do you think @qiuming-best @ywk253100 @reasonerjt

Reply:
There will be more and more breaking changes with time, and I wish we had a `BREAKING_CHANGE.md` file to declare these centrally. We then link it  in README.md